src/mango/test/16-index-selectors-test.py

```diff
@@ -161,6 +161,12 @@ def test_saves_partial_filter_selector_in_index(self):
         indexes = self.db.list_indexes()
         self.assertEqual(indexes[1]["def"]["partial_filter_selector"], selector)
 
+    def test_partial_filter_only_in_return_if_not_default(self):
```

Comment:
do we have a test for the negative as well (i.e. partial_filter_selector is not present in the json when it has the default value / was not specified)?